### ISSUE TITLE
Split up connectionArgs

### ISSUE THREAD

**Issue by taion** [graphql/graphql-relay-js#32]
Would it be possible to define `connectionArgs` per https://github.com/graphql/graphql-relay-js/blob/v0.3.2/src/connection/connection.js#L29 as e.g. the union of:

```js
export var connectionWithFirstArgs: GraphQLFieldConfigArgumentMap = {
  after: {
    type: GraphQLString
  },
  first: {
    type: GraphQLInt
  },
};

export var connectionWithLastArgs: GraphQLFieldConfigArgumentMap = {
  before: {
    type: GraphQLString
  },
  last: {
    type: GraphQLInt
  },
};

export var connectionArgs: GraphQLFieldConfigArgumentMap = {
  ...connectionWithFirstArgs,
  ...connectionWithLastArgs,
};
```

Per the spec: http://facebook.github.io/relay/graphql/connections.htm#sec-Arguments, connections don't have to implement both sets of arguments, so it'd be nice to have a helper directly from `graphql-relay-js` for defining just one pagination direction.

**Comment by taion** [comment]
Any thoughts on this? Obviously I don't need this in graphql-relay-js, but it seems like a useful helper to provide.

**Comment by wincent** [comment]
This seems reasonable to me. Thoughts @dschafer?

**Comment by taion** [comment]
You can't just add something like that in an edit :p I'll pull one together. Would appreciate better names, though.
